.chloggen/postgresql-bgwriter-duration-type.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Since this component is in `beta` do we need to handle this breaking change via the repo's breaking change policy?

Reply:
My understanding is that we adhere to the collector's contributing rules which include https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#breaking-changes

Reply:
I'm not sure it makes sense to apply any form of a two-step change here. Adding a new metric and deprecating this one seems problematic here. This change is arguably just a bug fix, but I marked it as breaking because technically it is possible that someone consumed the metadata or a few data points that successfully parsed (when the returned value happened to be a whole number).

Reply:
Ok I'm fine with that